best-practices/releasing_gems.md

```diff
@@ -25,3 +25,9 @@ $ gem owner base_indexer -a sul-devops-team@lists.stanford.edu
 ```
 
 The service account will ensure we all have access to our collective work. 
+
+## Steps to release a new version of a gem
+
+1. Update the version of the gem on the `master` branch according to the above [semantic versioning](http://semver.org) guidelines.
```

Comment:
Do all of our gems use `master` as the default branch? I haven't seen any that use, e.g., `develop`, so this is probably OK. I wonder about a small tweak here:

> 1. Update the version of the gem in the default branch (typically `master`) according to the above [semantic versioning](http://semver.org) guidelines

Reply:
Changed :-)

Reply:
THEN I APPROVE THIS PR EVEN HARDER